pkg/jobs/email.go

```diff
@@ -10,7 +11,9 @@ import (
 )
 
 const (
-	maxEmailAttempts = 2
+	maxEmailAttempts         = 2
```

Comment:
I think we should increase this as well, so allow for extra attempts if there is an influx of emails (e.g. someone creates an org and invites their team. 
```suggestion
	maxEmailAttempts         = 5
```